packages/knip/src/plugins/ladle/index.ts

```diff
@@ -0,0 +1,40 @@
+import type { EnablerPatterns } from '#p/types/config.js';
+import type { IsPluginEnabled, Plugin, ResolveEntryPaths } from '#p/types/plugins.js';
+import { hasDependency } from '#p/util/plugin.js';
+import type { LadleConfig } from './types.js';
+import { toEntryPattern } from '../../util/protocols.js';
+
+// https://ladle.dev/docs/config
+
+const title = 'ladle';
+
+const enablers: EnablerPatterns = [/^@ladle\//];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config = ['.ladle/config.{mjs,js,ts}'];
+
+const stories: string[] = ['**/*.@(stories.@(mdx|js|jsx|mjs|ts|tsx))'];
```

Comment:
Please use the same defaults as Ladle itself. Users can override in Knip config if necessary.